### PR TITLE
Updated names-generator.go

### PR DESCRIPTION
**Pull request by Morgy93**
Added some adjectives
Added some names

### CODE REVIEW

**Comment by Morgy93** [comment]
Oh my gosh, I am not able to sign the last commit, because it's always an automatic merge I MUST do even if I'm "Already up-to-date!".

**Comment by runcom** [comment]
@Morgy93 could you come on IRC and ping me? I'll help you get this done well :)

File: pkg/namesgenerator/names-generator.go [hunk 1]
```diff
@@ -189,12 +195,18 @@ var (
 
 		// Richard Feynman was a key contributor to quantum mechanics and particle physics. https://en.wikipedia.org/wiki/Richard_Feynman
 		"feynman",
+		
+		// Megan Denise Fox is an American actress and model. https://en.wikipedia.org/wiki/Megan_Fox
+		"fox",
```
Comment (crosbymichael) [inline pkg/namesgenerator/names-generator.go]: Transformers was a pretty awesome movie

**Comment by jfrazelle** [comment]
i think megan fox and jk rowling are cool buttttt idk if in the theme of the generator

**Comment by cpuguy83** [comment]
Not so sure about Satya either.... no offense to the guy!

**Comment by Morgy93** [comment]
I was wondering why this list is so tiny .. now I understand. 
If I was maintaining this, I would've taken some "most important men/women in history" list and added all of them. Just for the fun.
@jfrazelle How can some suspicious_fox or berserk_fox not be in the theme of the generator? =) And don't forget about the adoring_fox .. which is ambiguous .. somehow. =D

**Pull request closed without merging**